WorkflowUI/Sources/Hosting/WorkflowHostingController.swift

```diff
@@ -99,7 +99,7 @@ public final class WorkflowHostingController<ScreenType, Output>: UIViewControll
     }
 
     override public func viewDidLayoutSubviews() {
-        super.viewDidLayoutSubviews()
+        defer { super.viewDidLayoutSubviews() }
```

Comment:
Is this change needed for your hooks to work? We were considering moving the frame assignment to `viewWillLayoutSubviews` as part of the `ViewEnvironmentUI` feature, but wanted to isolate that change due to potential for side effects in code that was depending on this order of events.

Reply:
no i don't think this change is necessary. the previous implementation of `super.viewDidLayoutSubviews()` did nothing (according to the docs), and the new one emits an event. it seems like the event emission should occur after any logic in this method, but i don't think the ordering relative to the frame assignment will matter to any of the envisioned consumers.

if we're potentially going to move the frame assignment out of this method, then perhaps we should leave this proposed change since it will preserve the relative ordering with the new observation events?